MAUI/DataGrid/overview.md

```diff
@@ -47,4 +47,6 @@ The .NET MAUI DataGrid control is used to display and manipulate data in a tabul
 
 * **Exporting** - Support to export the data to Excel and PDF.
 
+ **Unbound Row** - It allows to add additional rows at top and also bottom of the SfDataGrid which are not bound with data object from underlying data source.
```

Comment:
It allows adding rows at the top and bottom of the DataGrid which are not bound with data objects from the underlying data source.

Reply:
Modified